apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -56,6 +62,31 @@ const ContributionsTable = ({
     setModals({ editContributionFormModal: true });
   };
 
+  type DialogProps = {
+    isOpen: boolean;
+    title : string;
+    onConfirm: boolean;
+    contribution_id: string;
+  }
+
+  const [dialog, setDialog] = useState<DialogProps>({ isOpen: false, title: '' ,onConfirm: false, contribution_id:''});
+ 
+  useEffect(() => {
+            setDialog(dialog)
+               }, [dialog])
+
+  
+  const handleDeleteContribution = (contribution_id: string) =>{
+    setDialog({
+      ...dialog,
+      isOpen: true,  //this opens AlertDialog
+      title: "Are you sure you want to delete this contribution? You can't undo this action afterwards",
```

Comment:
Let's change the title to: `"Are you sure you want to delete this Contribution? You can't undo this action."

We tend to capitalize Contribution thoughout the app

Reply:
Comment addressed. I have updated the title.